src/gleam/string.gleam

```diff
@@ -739,3 +739,70 @@ pub fn to_option(s: String) -> Option(String) {
     _ -> Some(s)
   }
 }
+
+/// Returns the first element in a grapheme and wraps it in an `Option(String)`.
```

Comment:
Sorry I'm not sure what this means. Does it return the first codepoint from the first grapheme of the string? Or the first grapheme?

Reply:
It turns out I might have misunderstood the terminology. I've updated the comments, so hopefully I got it right this time. Sorry!

Reply:
No problem